helm/wrongsecrets-ctf-party/values.yaml

```diff
@@ -92,18 +92,21 @@ balancer:
     REACT_APP_HEROKU_WRONGSECRETS_URL: "https://wrongsecrets-ctf.herokuapp.com" #required for 3 domain setup
     REACT_APP_CTFD_URL: "https://ctfd.io" #requierd for 2 and 3 domain setup
     REACT_APP_S3_BUCKET_URL: "s3://funstuff" #the s3 bucket you use for teh aws challenges, don't forget to make it accessible!
-    K8S_ENV: "k8s" #or 'aws', 'azure'
+    K8S_ENV: "gcp" #or 'aws', 'azure', or 'gcp'
```

Comment:
can we default it to k8s befor emerge please?

Reply:
yes. Done